enterprise/trackedusers/data_collector.go

```diff
@@ -0,0 +1,192 @@
+package trackedusers
```

Comment:
[subjective] I would go with mtu for package name. As it seems the dominant name used among business and engineering to refer to this feature. It is also short and unique.

```suggestion
package mtu
```